data/net/minecraft/core/Vec3i.mapping

```diff
@@ -56,27 +60,42 @@ CLASS net/minecraft/core/Vec3i
 		COMMENT Gets the Y coordinate.
 	METHOD getZ ()I
 		COMMENT Gets the Z coordinate.
+	METHOD lambda$static$0 ([I)Lnet/minecraft/core/Vec3i;
+		ARG 0 data
+	METHOD lambda$static$1 (Ljava/util/stream/IntStream;)Lcom/mojang/serialization/DataResult;
+		ARG 0 intStream
+	METHOD lambda$static$2 (Lnet/minecraft/core/Vec3i;)Ljava/util/stream/IntStream;
+		ARG 0 vec3i
 	METHOD multiply (I)Lnet/minecraft/core/Vec3i;
-		ARG 1 scalar
+		COMMENT Creates a new Vec3i with all values multiplied with the factor.
+		ARG 1 factor
+	METHOD north ()Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i one unit north.
 	METHOD north (I)Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i north by the distance.
 		ARG 1 distance
 	METHOD offset (DDD)Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i with the given offset. The values are getting floored.
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
 	METHOD offset (III)Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i with the given offset.
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
 	METHOD offset (Lnet/minecraft/core/Vec3i;)Lnet/minecraft/core/Vec3i;
-		ARG 1 vector
+		COMMENT Creates a new Vec3i with the given offset.
```

Comment:
This is technically incorrect, since if the given vector is (0,0,0) then it returns the same vector object.